docs/user/webapp/webapp.md

```diff
@@ -0,0 +1,102 @@
+# WebApp
+
+This guide explains how you can use the web app to create transformations. If you are using TOSCAna like descriped in [Getting Started](../getting-started.md) the WebApp application is already started and ready to use.
+
+
+## The start page
+![](img/start_page.png)
+*The start page without any CSARs*
+
+This page shows the default state of the app if no CSARs were added previously.
+
+## The top bar
+![](img/topbar.png)
+*The topbar*
+
+**Status:** The status shows the current state of the TOSCAna transformer, idling means he has nothing to do. If he is processing a transformation the status is _TRANSFORMING_.
+
+**Disk Space:** Disk space show how much server disk space is left.
+
+## CSARs
+
+### How to add a CSAR
+**1. Step:** Click on the ![](img/add_csar_button.png) button. After clicking on the button the app should look like in the image below.
+![](img/add_new_csar_bad.png)
+
+**2. Step:** Enter a name. If you leave the name empty and directly go to **Step 3** the name is automaticly filled in. If the name is a duplicate or something else is wrong the input field and the display indicator will be red. Hovering over the indicator will show the reason why the name is wrong
+
+**3. Step:** Click on the **Select file** button to upload a CSAR
+
+**4. Step:** Submit the CSAR
+
+### How to view a CSAR
+
+To view a CSAR click on it in the sidbar. After clicking, the web app should look similar to the image below. The current selected CSAR is highlighted blue in the sidebar.
+![](img/view_csars.png)
+*CSAR view for the lamp-input CSAR*
```

Comment:
Something is wrong with the layout.  There should be a line break between line 35 and 36.